underfs/obs/src/main/java/alluxio/underfs/obs/OBSConstants.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.underfs.obs;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * OBS constants.
+ */
+@ThreadSafe
+public final class OBSConstants {
+  public static final String HEADER_OBS = "obs://";
```

Comment:
Sorry one more thing!
Can you remove this file and put this header into `alluxio.Constants` together with other ufs like s3://

Reply:
done~